src/commands/api-mesh/create.js

```diff
@@ -122,6 +124,21 @@ class CreateCommand extends Command {
 
 		let shouldContinue = true;
 
+		if (
+			data?.meshConfig?.responseConfig?.includeHTTPDetails &&
+			workspaceName.toLowerCase() === 'production'
+		) {
+			this.warn(
+				`The mesh config has ${chalk.yellowBright('includeHTTPDetails')} set to ${chalk.redBright(
```

Comment:
Minor suggestion, but is there a convenient helper file we can move this to so that we keep logic and messaging consistent for this check on create and update?

Reply:
I dont think its a good idea to move the log prints to a helper function. We will have to send `this` for the helper function to print something to the console. Even if we create a new function to return the log message instead of printing, we are not getting much, we will still have to send the `data` object and `workspace` name. 

Its ok to localize conditionals and log prints to the commands.

Reply:
Understandable. I'm a stickler for minimizing duplicate code for the sake of consistency and maintainability. Doesn't seem worth the effort here though.